client/batch.go

```diff
@@ -308,6 +315,28 @@ func (b *Batch) Scan(s, e interface{}, maxRows int64) {
 	b.initResult(1, 0, nil)
 }
 
+// ReverseScan retrieves the rows between begin (exclusive) and end (inclusive).
```

Comment:
For clarity, I'd rewrite this as: "in reserve order starting at end (inclusive) and finishing at begin(exclusive)."